build/deprecated/build-utils/Cargo.toml

```diff
@@ -11,4 +11,5 @@ serde = { version = "1.0", features = ["derive"] }
 
 [dependencies.reqwest]
 version = "0.11.12"
-features = ["blocking", "json"]
+default-features = false
+features = ["blocking", "json", "rustls-tls"]
```

Comment:
You should be able to get rid of this whole section altogether and use something like 
```
reqwest = { workspace = true }
```
So we keep settings in one place.